openshift/dynamic/client.py

```diff
@@ -15,11 +15,21 @@
 
 from openshift.dynamic.exceptions import ResourceNotFoundError, ResourceNotUniqueError, api_exception, KubernetesValidateMissing
 
+
+
 try:
     import kubernetes_validate
     HAS_KUBERNETES_VALIDATE = True
+    try:
+        class VersionNotSupportedError(kubernetes_validate.utils.VersionNotSupportedError):
```

Comment:
Would it make more sense to do an import here, rather than redefining the class?